mobile/ios/ios-emulator-services.ts

```diff
@@ -93,3 +106,5 @@ class IosEmulatorServices implements Mobile.IEmulatorPlatformServices {
 	}
 }
 $injector.register("iOSEmulatorServices", IosEmulatorServices);
+
+export = IosEmulatorServices;
```

Comment:
Yes, this will be equivalent of ```module.exports.IosEmulatorService = IosEmulatorService```.
I've checked your other PR in order to see why you need the export to be this way and it look like you just need the type. My suggestion is to use 
```TypeScript
export class IosEmulatorService implements Mobile.IiOSEmulatorPlatformServices
```
where IiOSEmulatorPlatformService is :
```TypeScript
interface IiOSEmulatorPlatformService extends IEmulatorPlatformServices {
    postDarwinNotification(notification: string): IFuture<void>;
}
```
And in https://github.com/NativeScript/nativescript-cli/pull/479/files#diff-3ea8e9be6fce663fc286a4462786a297R48 you can use 
```TypeScript
private $iOSEmulatorServices: Mobile.IiOSEmulatorPlatformService
```
Maybe I'm missing something why you need the export to be ```module.exports = IosEmulatorService```